STEERING-COMMITTEE.md

```diff
@@ -162,13 +120,157 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee will have 7 seats.  Five of the seats will be
+Contributing seats, open to any project contributor.  Two of the seats will be
+End-User seats, open only to active end-user community members.  See Candidate
+Eligibility for definitions of these categories.
+
+Steering Committee members will serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats will be
+elected, and one end user seat will be elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections shall be held in October or November of each year.
```

Comment:
Do we want to suggest that this is intentionally about 6 months off from the TOC to allow better knowledge transfer and continuity?

Reply:
The material on the TOC elections currently doesn't say what time of year it will be held, I wonder if we should just say "annually" ? WDYT?

Reply:
I'd leave it and update TOC.  It'd be good to indicate when that happens for folks looking to understand how they might participate.